tests/h/services/search_index/_queue_test.py

```diff
@@ -351,6 +338,14 @@ def index(annotations):
 
         return index
 
+    @pytest.fixture
+    def add_all(self, queue, annotation_ids):
+        def add_all(ids=annotation_ids, schedule_in=MINUS_5_MIN_IN_SECS, force=False):
+            for id_ in ids:
+                queue.add(id_, tag="test_tag", schedule_in=schedule_in, force=force)
+
+        return add_all
```

Comment:
This is needed for the obvious reason that `add_all` is gone, but also for the slightly less obvious reason that if you do the "correct" way here:

```queue.add_where(where=[Annotation.id.in_(annotation_ids), ...```

Then the function naturally dedupes the jobs. This is a good feature, but our test often want to setup multiple jobs.